### PR TITLE
[Dev] Adding stories for Messaging

### LINKED ISSUE #592: [MSG] Add stories for Inbox and Compose views
📖 time!

### PR DESCRIPTION
**Pull request by mennenia**
Fix #592 

Keeping the stories fairly simple here, as I'll be devving story-first, they will expand over time ^_^.

Some backstory about using `<StubContainer />` - Similar to writing tests, I wanted to be able to write stories with dummy data, but seeing as the `Inbox` is a `RootContainer` relying on Relay, just passing in some props didn't work. @orta mentioned the stub containers as they're used in the Artist components also and they're great for quick stories!

### CODE REVIEW

**Comment by ArtsyOpenSource** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Fails</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:no_entry_sign:</td>
      <td>

  No CHANGELOG added.
  </td>
    </tr>
  </tbody>
</table>


<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Warnings</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:warning:</td>
      <td>

  Please assign someone to merge this PR, and optionally include people who should review.
  </td>
    </tr>
  </tbody>
</table>



<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

File: src/lib/components/inbox/conversations/__stories__/composer.story.tsx [hunk 1]
```diff
@@ -0,0 +1,7 @@
+import { storiesOf } from "@storybook/react-native"
+import * as React from "react"
+import "react-native"
+import Composer from "../composer"
+
+storiesOf("Conversations - Composer")
+  .add("With no preexisting thread", () => <Composer />)
```
Comment (orta) [inline src/lib/components/inbox/conversations/__stories__/composer.story.tsx]:
```diff
- preexisting
+ pre-existing
```

File: src/lib/components/inbox/conversations/__stories__/conversation-snippet.story.tsx [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+        href: "/artwork/bradley-theodore-karl-and-anna-face-off-diptych",
+        title: "Karl and Anna Face Off (Diptych)",
+        date: "2016",
+        artist: {
+        name: "Bradley Theodore",
+        },
+        image: {
+        url: "https://d32dm0rphc51dk.cloudfront.net/bJ9I_vJX9ksaKFJAkOAIKg/normalized.jpg",
+        image_url: "https://d32dm0rphc51dk.cloudfront.net/bJ9I_vJX9ksaKFJAkOAIKg/:version.jpg",
+        },
```
Comment (orta) [inline src/lib/components/inbox/conversations/__stories__/conversation-snippet.story.tsx]: this isn't formatted, should be fine #567 will fix in the future

File: src/lib/components/inbox/conversations/__stories__/conversation-snippet.story.tsx [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+  ],
+}
+
+storiesOf("Conversations - Snippet")
+  .add("Single row", () => <ConversationSnippet conversation={conversation} />)
+  .add("A few threads", () => <ScrollView>
+      <ConversationSnippet conversation={conversation} />
+      <ConversationSnippet conversation={conversation} />
+      <ConversationSnippet conversation={conversation} />
+    </ScrollView>)
```
Comment (orta) [inline src/lib/components/inbox/conversations/__stories__/conversation-snippet.story.tsx]: This is nice

**Pull request merged**